conductr_cli/sandbox.py

```diff
@@ -97,6 +103,15 @@ def add_default_arguments(sub_parser):
     sub_parser.add_argument('--local-connection',
                             default=True,
                             help=argparse.SUPPRESS)
+    add_resolve_ip(sub_parser, True)
+
+
+def add_resolve_ip(sub_parser, default_value):
+    sub_parser.add_argument('--resolve-ip',
+                            default=default_value,
+                            dest='resolve_ip',
+                            action='store_true',
+                            help=argparse.SUPPRESS)
```

Comment:
This is a switch to disable the attempt to validate ConductR IP obtained from `docker-machine`,  `boot2docker`, or environment variable.